### PR TITLE
Refine figures

### PR DESCRIPTION
**Pull request by x-tabdeveloping**
Made figures nicer for the paper.

### CODE REVIEW

**Comment by x-tabdeveloping** [comment]
Also added results from the benchmark.

File: topic_benchmark/figures.py [hunk 1]
```diff
@@ -19,87 +19,106 @@
...
+        y="Relative Frequency of Stop Words",
         template="plotly_white",
         category_orders=CATEGORY_ORDERS,
-        facet_col="encoder",
+        facet_col="Encoder",
         color_discrete_sequence=px.colors.qualitative.Pastel,
     )
     fig = fig.update_traces(line=dict(width=3))
-    fig = fig.update_layout(width=1000, height=800)
+    fig = fig.update_layout(width=900, height=400)
```
Comment (jankounchained) [inline topic_benchmark/figures.py:90]: suggesting to have text in black, more readable, better looking

`fig = fig.update_layout(width=900, height=400, font_color="black")`

File: topic_benchmark/figures.py [hunk 2]
```diff
@@ -19,87 +19,106 @@
...
+        y="Relative Frequency of Nonalphabetical Terms",
         category_orders=CATEGORY_ORDERS,
-        facet_col="encoder",
+        facet_col="Encoder",
         template="plotly_white",
         color_discrete_sequence=px.colors.qualitative.Pastel,
     )
     fig = fig.update_traces(line=dict(width=3))
-    fig = fig.update_layout(width=1000, height=800)
+    fig = fig.update_layout(width=1400, height=600)
```
Comment (jankounchained) [inline topic_benchmark/figures.py:107]: suggesting to have text in black, more readable, better looking

`fig = fig.update_layout(width=900, height=400, font_color="black")`

File: topic_benchmark/figures.py [hunk 3]
```diff
@@ -19,87 +19,106 @@
...
+        facet_row="Dataset",
+        facet_col="Encoder",
+        y="Runtime in Seconds",
         template="plotly_white",
         color_discrete_sequence=px.colors.qualitative.Pastel,
         category_orders=CATEGORY_ORDERS,
     )
     fig = fig.update_traces(line=dict(width=3))
-    fig = fig.update_layout(width=1000, height=800)
+    fig = fig.update_layout(width=1000, height=600, margin=dict(b=0))
```
Comment (jankounchained) [inline topic_benchmark/figures.py:123]: suggesting to have text in black, more readable, better looking

`fig = fig.update_layout(width=900, height=400, font_color="black")`

**Review by jankounchained — COMMENTED** [review]
suggesting to have text in black, more readable, better looking

`fig = fig.update_layout(width=900, height=400, font_color="black")`

**Pull request merged**